app/views/moderation/images/index.html.haml

```diff
@@ -1,6 +1,13 @@
 %h1 Images externes les plus récentes
 
-- @images.each do |image|
-  .image
-    = image.to_html.html_safe
-    = button_to "Bloquer", moderation_image_path(image.encoded_link), method: :delete, data: { confirm: "Confirmez‑vous vouloir bloquer cette image ?" }
+- if @images.any?
+  %p Images externes les plus récentes
+  - @images.each do |image|
+    .image
+      = image.to_html.html_safe
+      - if image.blocked
+        = "⛔ Bloquée" if image.blocked
+      - else
+        = button_to "Bloquer", moderation_image_path(image.encoded_link), method: :delete, data: { confirm: "Confirmez‑vous vouloir bloquer cette image ?" }
```

Comment:
Ok, that's better, thanks :)

I've just noticed this issue while testing this code:

1. As a moderator, I open the moderation image list
2. I view an image named "A"
3. I block it (using the "Bloquer" button and confirm with the popup)
4. I force Firefox to reload the moderation image list
5. Now the image "A" is blocked and so the image service ("img.dlfp.lo") responds with a 404 File Not found
6. As a moderator, I see the blocked URL rendered as an image (`<img>`), so I cannot copy / paste link easily to check the content.

For exemple, on this screenshot, I cannot select URIs, because they are `<img>` HTML tags rendering the `alt` attribute as an image:
![image](https://github.com/linuxfrorg/linuxfr.org/assets/307130/9f3f4b73-239e-48bc-9492-9a24255cbd2d)

To workaround this issue (with at least Firefox), I would do the same you've made on admin image list and use a `<pre>` HTML tag.

```suggestion
- if @images.any?
  %p Images externes les plus récentes
  %ul
    - @images.each do |image|
      %li.image
        - if image.blocked
          %pre #{image.link.html_safe}
          = "⛔ Bloquée"
        - else
          = image.to_html.html_safe
          = button_to "Bloquer", moderation_image_path(image.encoded_link), method: :delete, data: { confirm: "Confirmez‑vous vouloir bloquer cette image ?" }
```